### PR TITLE
Apr 1337 subcontractor declaration

### PR DESCRIPTION
**Pull request by MichaelYoung1981**

### CODE REVIEW

File: src/SFA.DAS.ApplyService.InternalApi/Infrastructure/InternalQnaApiClient.cs [hunk 1]
```diff
@@ -12,7 +14,7 @@
 
 namespace SFA.DAS.ApplyService.InternalApi.Infrastructure
 {
-    public class InternalQnaApiClient:IInternalQnaApiClient
+    public class InternalQnaApiClient : IInternalQnaApiClient
```
Comment (David-B-Read) [inline src/SFA.DAS.ApplyService.InternalApi/Infrastructure/InternalQnaApiClient.cs]: As we're adding more and more functionality into the QnA API Client used by the Internal API, would it be better to share the one thats in ApplyService.Web in an Api Clients project?

Comment (MichaelYoung1981) [reply src/SFA.DAS.ApplyService.InternalApi/Infrastructure/InternalQnaApiClient.cs]: Good question.  There is currently only one common method between the two classes.  Hows about the next time someone adds a method that already exists on the one in ApplyService.Web we get them to abstract out the common methods? 😁

Comment (David-B-Read) [reply src/SFA.DAS.ApplyService.InternalApi/Infrastructure/InternalQnaApiClient.cs]: GetPageBySectionNo, GetQuestionTag, GetAnswerValue, DownloadFile are all implemented in the ApplyService.Web client.

Comment (MarkFCain) [reply src/SFA.DAS.ApplyService.InternalApi/Infrastructure/InternalQnaApiClient.cs]: Yeah the ones in this are in both.  I don't know whether a common catchall one is better cos internal doesn't need all the various calls in the web version.  It's more like take the internal one, make it higher level (own project), and remove the same from the web one and repoint any usages of those methods to the higher level one.. I think this makes sense.

**Comment by David-B-Read** [comment]
Merged in https://github.com/SkillsFundingAgency/das-apply-service/pull/529

**Pull request closed without merging**